cmd/crates/soroban-rpc/src/lib.rs

```diff
@@ -21,15 +21,20 @@ use std::{
     str::FromStr,
     time::{Duration, Instant},
 };
+use stellar_xdr::curr::ContractEventType;
 use termcolor::{Color, ColorChoice, StandardStream, WriteColor};
 use termcolor_output::colored;
 use tokio::time::sleep;
 
-use crate::utils::contract_spec;
-
+pub mod log;
 mod txn;
 
+pub use txn::*;
```

Comment:
Looks like we're only using `txn::Assembled` so would `pub use tnx::Assembled` be better so we're not pulling everything in? Disclaimer: I'm not a Rust expert

Reply:
It won't cause the dependent crate to bloat since rust has tree shaking by default. However, you raise a good point: since this crate is now a library, it's probably best to limit the API and the blanket * could cause issues in the future if anything is added to `txn`.